drivers/hmis/app/graphql/mutations/create_assessment.rb

```diff
@@ -28,6 +30,7 @@ def resolve(enrollment_id:, assessment_role:, assessment_date: nil)
         data_source_id: user.data_source_id,
         user_id: user.user_id,
         personal_id: enrollment.personal_id,
+        enrollment_id: enrollment.id,
```

Comment:
This mutation was throwing exceptions for me in graphiql, and I figured out it's because this should be:
```suggestion
        enrollment_id: enrollment.enrollment_id,
```

You can replicate the failure in the tests by replacing `sequence(:EnrollmentID, 1)` with `sequence(:EnrollmentID, 500)` (or UUID or something else) in the `hmis_hud_enrollment` factory. This was allowing the error to slip by because `id` and `EnrollmentID` were both 1.